app/components/edit_channel_info/index.js

```diff
@@ -0,0 +1,422 @@
+                                    autoCapitalize='none'
+                                    autoCorrect={false}
+                                    placeholder={{id: 'channel_modal.headerEx', defaultMessage: 'E.g.: "[Link Title](http://example.com)"'}}
+                                    placeholderTextColor={changeOpacity('#000', 0.5)}
+                                    multiline={true}
+                                    blurOnSubmit={false}
+                                    onFocus={this.scrollToEnd}
+                                    textAlignVertical='top'
+                                    underlineColorAndroid='transparent'
+                                />
+                            </View>
+                            <View ref={this.lastTextRef}>
+                                <FormattedText
+                                    style={style.helpText}
+                                    id='channel_modal.headerHelp'
+                                    defaultMessage={'Set text that will appear in the header of the channel beside the channel name. For example, include frequently used links by typing [Link Title](http://example.com).'}
+                                />
+                            </View>
+                        </View>
+                    </TouchableWithoutFeedback>
+                </KeyboardAwareScrollView>
+            </View>
+        );
+    }
+}
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => {
+    return {
+        container: {
+            flex: 1,
+            backgroundColor: theme.centerChannelBg
+        },
+        scrollView: {
+            flex: 1,
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.03),
+            paddingTop: 10
+        },
+        errorContainer: {
+            backgroundColor: changeOpacity(theme.centerChannelColor, 0.03)
+        },
+        errorWrapper: {
+            justifyContent: 'center',
+            alignItems: 'center'
+        },
+        inputContainer: {
+            marginTop: 10,
+            backgroundColor: '#fff'
+        },
+        input: {
+            color: '#333',
+            fontSize: 14,
+            height: 40,
+            paddingHorizontal: 15
+        },
+        titleContainer30: {
+            flexDirection: 'row',
+            marginTop: 30
+        },
+        titleContainer15: {
+            flexDirection: 'row',
+            marginTop: 15
+        },
+        title: {
+            fontSize: 14,
+            color: theme.centerChannelColor,
+            marginLeft: 15
+        },
+        optional: {
+            color: changeOpacity(theme.centerChannelColor, 0.5),
+            fontSize: 14,
+            marginLeft: 5
+        },
+        helpText: {
+            fontSize: 14,
+            color: changeOpacity(theme.centerChannelColor, 0.5),
+            marginTop: 10,
+            marginHorizontal: 15
+        },
+        navTitle: {
```

Comment:
where is this being used?

Reply:
going to remove that. It was passed over from the create_channel component. I'll verify styles all correspond to the components